build.gradle

```diff
@@ -400,6 +396,17 @@ uploadArchives {
                 ' for more information.' )
             throw new Exception("Missing sonatype username or password, please see the README for properties that must be specified for uploading an archive.")}
         )
-
     }
+
+distributions {
+    spark{} //added to allow an installSparkDist target which will also build the spark jar
+}
+
+installSparkDist.dependsOn downloadGsaLibFile, sparkJar
```

Comment:
So as far as I understand, the gradle local ablation default installDist, because the early maven plug in stole install.